invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/frontpage.less

```diff
@@ -72,26 +73,189 @@ only screen and (max-width: 993px) {
     flex-direction: row;
 }
 
-.goto-button {
-    text-align: center;
-    padding: 20px 10px;
-    white-space: nowrap;
-}
-
 .goto-button a {
     text-decoration: none;
-    border-radius: 4px;
-    line-height: 25px;
-    padding: 15px 30px;
-    background-color: #0377cd;
-    color:  white !important;
-    height: 50px;
+    color:  white;
     font-size: 1em;
-    font-weight: 200;
+    font-weight: 600;
 }
 
 .goto-button a:hover,
 .goto-button a:active {
-    color: white !important;
-    background-color: #01599c;
+    color: white;
+}
+
+.section-content-dark-bg,
+.section-content-white-bg,
+.section-content-light-bg {
+  padding-top: calc(30px);
```

Comment:
We should use em/rem values instead of px to make sure the spacing is scaling well with the browser's font-size setting. (For this to work as expected everywhere, we also need to replace the `font-size: 14px` from the  html- and body-element with a rem value.)

Reply:
yes I plan to refactor this too, but in the next iteration, for now I am removing redundant files